File: app/src/main/java/ua/com/amicablesoft/android/wr/ui/BaseTitleLessDialogFragment.java
package ua.com.amicablesoft.android.wr.ui;

import android.app.Dialog;
import android.app.DialogFragment;
import android.os.Bundle;
import android.view.Window;

/**
 * Created by olha on 12/18/16.
 */

public class BaseTitleLessDialogFragment extends DialogFragment {

    @Override
    public Dialog onCreateDialog(Bundle savedInstanceState) {
        Dialog dialog = super.onCreateDialog(savedInstanceState);

        // request a window without the title
        dialog.getWindow().requestFeature(Window.FEATURE_NO_TITLE);
        dialog.setCancelable(false);
        dialog.setCanceledOnTouchOutside(false);
        setCancelable(false);
        return dialog;
    }
}

File: app/src/main/java/ua/com/amicablesoft/android/wr/ui/processing/BaseTitleLessDialogFragment.kt
/*
 * Created by Olha Tymoshenko on 5/25/18 3:03 PM.
 * Copyright (c) 2018. All rights reserved.
 */

package ua.com.amicablesoft.android.wr.ui.processing

import android.app.Dialog
import android.app.DialogFragment
import android.os.Bundle
import android.view.Window

open class BaseTitleLessDialogFragment : DialogFragment() {

    override fun onCreateDialog(savedInstanceState: Bundle): Dialog {
        val dialog = super.onCreateDialog(savedInstanceState)

        // request a window without the title
        dialog.window!!.requestFeature(Window.FEATURE_NO_TITLE)
        dialog.setCancelable(false)
        dialog.setCanceledOnTouchOutside(false)
        isCancelable = false
        return dialog
    }
}
